packages/block-editor/src/components/link-control/style.scss

```
@@ -184,13 +184,14 @@ $preview-image-height: 140px;
 		position: relative;
 		top: 0.2em;
 		margin-right: $grid-unit-10;
-		width: 18px; // half of 32px to improve perceived resolution.
-		height: 18px; // half of 32px to improve perceived resolution.
+		max-height: 24px;
+		flex-shrink: 0;
+		width: 24px;
+		display: flex;
+		justify-content: center;
 
-		svg,
 		img {
-			max-width: none;
-			width: 18px; // half of 32px to improve perceived resolution.
+			width: 16px;
```

```suggestion
			width: 16px; // downscale image x2 improve perceived resolution of smaller favicons.
```

Nit: I'd like to restore this to make it explicit as to what we are doing and _why_.

---

Have we tried getting larger icons through the `apple-touch-icon` property instead of going straight to favicon?

---

It is on the roadmap...but you cannpt guarantee it. Therefore we still need these aa defaults for now.